src/key/private_key.js

```diff
@@ -83,6 +80,9 @@ class PrivateKey extends PublicKey {
     const primaryKey = this.keyPacket;
     const keys = [];
     for (let i = 0; i < this.subkeys.length; i++) {
+      if (helper.isPublicOrDummyKeyPacket(this.subkeys[i].keyPacket)) {
+        continue;
+      }
```

Comment:
While this is consistent with the existing behavior, I think we could improve the error reporting by keeping this in `decryptSessionKeys`...